src/mongo_client.ts

```diff
@@ -509,6 +509,13 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> implements
   /**
    * Connect to MongoDB using a url
    *
+   * @remarks
+   * Calling `connect` is optional since the first operation you perform will call `connect` if it's needed.
+   * `timeoutMS` specified at the client-level will bound the time any operation can take before throwing a timeout error.
```

Comment:
```suggestion
   * `timeoutMS` will bound the time any operation can take before throwing a timeout error.
```

This is true, regardless of where timoeutMS is is specified.